blogging/blog_guidelines.md

```diff
@@ -0,0 +1,131 @@
+
+ - [Blog preparation](#blog-preparation)
+ - [Publishing steps](#publishing-steps)
+   - [When ready to write](#when-ready-to-write)
+   - [When ready to publish](#when-ready-to-publish)
+ - [Content Guidelines](#content-guidelines)
+   - [Zowe Core](#zowe-core)
+   - [Zowe Ecosystem](#zowe-ecosystem)
+   - [Zowe Incubation](#zowe-incubation)
+ - [Personas](#personnas)
+ - [Promotion](#promotion)
+
+## Blog preparation
+
+**Blogs should:**
+
+ - Be brief (generally < 1200; > 500 words)
+ - Focus on a target audience
+ - Be relevant
+   - The title and first paragraph should compel the viewer to take the time to read the piece.  The content should be relevant and helpful for the target audience.
+ - Leverage visuals (screenshots, GIFs, etc.), especially an eye-catching featured image  
+   - See public, license-free sites like unsplash.com
+
+**Discoverability (search results):**
+
+ - Determine which Google search keywords are best suited for your content and target audience (i.e., what are you communicating and to whom?  What keywords are they likely to use?).  Include these keywords in your title and related words in the body, especially the first paragraph.
+
+ - Be sure your title, opening paragraph (and featured image) will stand out in a search by your target audience (i.e., will it get people to click your entry over all the other Google results?)
+
+## Publishing steps
+
+ - Create a Medium handle.  
+ - Let [zowe-blog-editors@openmainframeproject.org](email:zowe-blog-editors@openmainframeproject.org) know your handle so you can be added as a “writer” for the publication.  To see your handle, select your profile picture when you log into medium.com, it will begin with an `@` symbol, e.g. `@myusername_1974.  
+
+### When ready to write
+
+ - Go to [Medium.com](https://medium.com)
+ - Click your profile (should be logged in)
+ - Click New Story
+ - Begin editing
+ - If needed, save as “draft”
+ - Include the following at the end of your blog: [See more Zowe blogs here](https://medium.com/zowe)
+ - Include 5 tags like: “Zowe”, “Mainframe”, “DevOps”, “Open Source”, [specific technology like COBOL, vscode], [specific roles like Sysprog], etc.
+
+### When ready to publish
+
+ - You may need to add the publication to your profile (Profile => Publication => New Publication)
+ - Double check: be sure you have...
+   - compelling title and opening paragraph (discoverability is critical!)
+   - featured image and visuals in place.  By default, the first image will be associated with the blog post, however you can choose to use a different headline image by navigating to the image while in edit mode, select it manually, and press `Shift + F`. 
+   - appropriate links and tags
+ - While editing the story, click “…”
+ - Click “Add to Publication”
+ - If applicable, uncheck “Allow curators to recommend my story to interested readers.”
+ - Click “Submit to Publication”
+ - The blog will remain Pending until the publication owner posts it
+
```

Comment:
## Review Process:
- 2 members of the project from different companies will be assigned to review for technical details
- OMP team will do final check for grammar and branding
- OMP will scheduled it and promote via social
- OMP team will let author know publish date and share social media
Please do not publish your own post.